WatchApp Extension/Scenes/GlucoseChartScene.swift

```diff
@@ -204,18 +205,19 @@ class GlucoseChartScene: SKScene {
     }()
 
     override init() {
-        // Use the fixed sizes specified in the storyboard, based on our guess of the model size
+        // Set the height based on the model size, which we guess from the screen width
         let screen = WKInterfaceDevice.current().screenBounds
-        let width = screen.width - 2 /* insets */
         let height: CGFloat
 
-        switch width {
-        case let x where x < 150:  // 38mm
-            height = 68
-        case let x where x > 180:  // 44mm
-            height = 106
-        default:
-            height = 86
+        switch screen.width {
```

Comment:
This should revert to the previous switch statement, which is based on how asset catalogs work; handle the 38 and the 44 as custom, and default to 42/40 which are nearly the same screen size.

Also the added comments are wrong; "162x197 px" is points, not pixels (the screen is @2x)